docker-compose.yml

```diff
@@ -2,10 +2,7 @@ version: '3.7'
 
 services:
   app:
-    image: yt-spammer-purge
-    build:
```

Comment:
Keep the build option for local development and testing

Reply:
I added it back within a docker-compose override file.
The two options kind of conflict with each other.
Let me know what you think.